apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -23,10 +30,23 @@ const Analytics = (): JSX.Element => {
 		[handleSubmit],
 	);
 
+	const handleContributorsSearchChange = useCallback(
+		(value: string) => {
+			onSearch(value);
+		},
+		[onSearch],
+	);
+
 	return (
 		<PageLayout>
 			<h1 className={styles["title"]}>Analytics</h1>
-			<section>
+			<section className={styles["filters"]}>
+				<AnalyticsContributorsSearch
+					control={control}
+					errors={errors}
+					name="search"
+					onChange={handleContributorsSearchChange}
+				/>
```

Comment:
Shouldn't search component be inside <form> as well as DataInput?

Reply:
I believe that yes, `Search` and `DateInput` should ideally be inside `<form>`, thanks :)